ycmd.el

```diff
@@ -1462,6 +1468,22 @@ Otherwise the response is probably an exception."
     (ycmd--report-status 'parsed)
     (run-hook-with-args 'ycmd-file-parse-result-hook results)))
 
+(defun ycmd--notify-server (event-name)
+  "Send a simple event notification to the server, ignoring response."
```

Comment:
`event-name` needs to be mentioned in the docstring with capital letters i.e. `EVENT-NAME`